wg-advisors/advisors.md

```diff
@@ -0,0 +1,14 @@
+Here you will find a list of advisors who signed up to be listed in this repository.
+
+In order to be listed here, you must be an ASF member. If you are one, you can just
+open a PR to add yourself to this list. If you are listed here and you no longer want
+to be an advisor, please remove yourself with a PR.
+
+If you want to advise a PMC, please reach out to the PMC and ask if they would like to have you as an advisor
```

Comment:
Link back to "How to initiate PMC <-> Advisor relationship?".

Reply:
Updated, squashed and rebased.